src/main/java/org/apache/maven/buildcache/xml/CacheConfigImpl.java

```diff
@@ -452,28 +453,29 @@ public boolean isEnabled() {
     @Override
     public boolean isRemoteCacheEnabled() {
         checkInitializedState();
-        return getRemote().getUrl() != null && getRemote().isEnabled();
```

Comment:
Though nothing changed here, the `RemoteCacheEnabled` flag intuitively should disable all the remote interactions. So all the flags below should be preconditioned on the `remote. enabled`. Again, there is no regression in the current form, and this is an optional consideration.

Reply:
Added